SmartDeviceLink/SDLLifecycleManager.m

```diff
@@ -843,6 +918,19 @@ - (void)videoServiceProtocolDidUpdateFromOldProtocol:(nullable SDLProtocol *)old
     }
 }
 
+- (void)encryptionServiceProtocolDidUpdateFromOldProtocol:(nullable SDLProtocol *)oldProtocol toNewProtocol:(nullable SDLProtocol *)newProtocol {
```

Comment:
This seems odd to me. Why wouldn't it just be stopped when the lifecycle manager stops, and started when it's started again?